hedera-node/hedera-mono-service/src/main/java/com/hedera/services/store/contracts/precompile/impl/TransferPrecompile.java

```diff
@@ -316,6 +321,15 @@ public long getMinimumFeeInTinybars(final Timestamp consensusTime) {
             accumulatedCost += transfer.fungibleTransfers().size() * ftTxCost;
             accumulatedCost += transfer.nftExchanges().size() * nonFungibleTxCost;
         }
+
+        // add the cost for transferring hbars
```

Comment:
Q: Is there a  cost capturing the approval transfer.
For example do we capture the work done by the network to verify the approved transfer is permissible?

Reply:
I do not see this use case called out in the [fee schedule](https://docs.hedera.com/guides/mainnet/fees#transaction-and-query-fees)  Nor do I see any extra fees being charged for approved transfers in HAPI.  There is a fee for setting the approval in the first place.